addons/web_editor/models/assets.py

```diff
@@ -273,3 +273,29 @@ def _save_asset_view_hook(self):
             dict
         """
         return {}
+
+    def _make_scss_customization(self, url, values):
```

Comment:
What this method does is really specific to website, that's why it was in a website controller before. As many times before: you should leave this in website, not put it in web_editor.

I think there already is a customization of web_editor.assets in website right ? You can add it there.

Also, if the method is meant to be called by the controller, it should not be private (and so it should not start by `_`). Also name the method `make_website_scss_customization` to be clearer.